### ISSUE TITLE
Add plugin via CLI issues: not found & supported platforms

### ISSUE THREAD

**Issue by TanaseButcaru** [MobileChromeApps/cordova-plugin-crosswalk-webview#26]
When I try to add the plugin via cordova CLI I have the following issues:

(1) ``cordova plugin add cordova-plugin-crosswalk-webview`` returns the following error:
```
Fetching plugin "cordova-plugin-crosswalk-webview" via plugin registry
npm http GET http://registry.cordova.io/cordova-plugin-crosswalk-webview
npm http 404 http://registry.cordova.io/cordova-plugin-crosswalk-webview
Error: 404 Not Found: cordova-plugin-crosswalk-webview
    at RegClient.<anonymous> (/usr/lib/node_modules/cordova/node_modules/cordova-lib/node_modules/npm/node_modules/npm-registry-client/lib/request.js:268:14)
    at Request.self.callback (/usr/lib/node_modules/cordova/node_modules/cordova-lib/node_modules/npm/node_modules/request/index.js:148:22)
    at Request.emit (events.js:110:17)
    at Request.<anonymous> (/usr/lib/node_modules/cordova/node_modules/cordova-lib/node_modules/npm/node_modules/request/index.js:876:14)
    at Request.emit (events.js:129:20)
    at IncomingMessage.<anonymous> (/usr/lib/node_modules/cordova/node_modules/cordova-lib/node_modules/npm/node_modules/request/index.js:827:12)
    at IncomingMessage.emit (events.js:129:20)
    at _stream_readable.js:908:16
    at process._tickCallback (node.js:355:11)
```
...which is right, because the plugin cannot be found <a href="http://registry.cordova.io/cordova-plugin-crosswalk-webview">here</a>.

(2) The plugin is added for all project's platforms (in my case android & ios). It doesn't break anything on iOS, but still...
```
cordova plugin add ../cross
Installing "cordova-plugin-crosswalk-webview" for android
Installing "cordova-plugin-crosswalk-webview" for ios
```

**Comment by agrieve** [comment]
This is an unfortunate temporary situation. As soon as the next version of Cordova's CLI is released (any time now), it will work.

As for installing on iOS, this is a limitation of Cordova's plugin tooling, where right now it doesn't know to do nothing for platforms not supported for a plugin.

**Comment by TanaseButcaru** [comment]
Solved after cordova CLI 5.0 update.
